sources/packages/web/README.md

```diff
@@ -1,28 +1,23 @@
-# web
+# vue-prime
 
 ## Project setup
 ```
-npm install
+yarn install
```

Comment:
Any particular reason to change dependency manager to yarn?

Reply:
Hi @fwpushan, I did a new commit to remove Yarn and use npm.
Yarn was added by mistake during the creation of the new Vue 3 solution.